src/textview/textview.ios.ts

```diff
@@ -232,6 +243,10 @@ export class TextView extends TextViewBase {
     [floatingColorProperty.setNative](value: Color) {
         const color = value instanceof Color ? value.ios : value;
         this._controller.floatingPlaceholderActiveColor = color;
+    }
+    [floatingInactiveColorProperty.setNative](value: Color) {
+        const color = value instanceof Color ? value.ios : value;
+        this._controller.inlinePlaceholderColor = color;
```

Comment:
here you added `inlinePlaceholderColor` what is it for? Why is it set in `floatingInactiveColorProperty` and not `floatingColorProperty`?

Reply:
See other comment, same reasoning